draft-ietf-rtcweb-jsep.xml

```diff
@@ -2840,8 +2840,13 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <xref target="RFC4145"></xref>, Section 4, and clarified
               for use in DTLS-SRTP scenarios in
               <xref target="RFC5763"></xref>, Section 5. The role value
-              in the answer MUST be "active" or "passive"; the "active"
-              role is RECOMMENDED.</t>
+              in the answer MUST be "active" or "passive", and MUST
+              be consistent with the value in the offer. Where
```

Comment:
I think the best thing to say would be that when "actpass" is presented, "active" is RECOMMENDED. Otherwise, replying with the correct value is a MUST.